### ISSUE TITLE
NVDA continues to voice suggestions in address bar even when focus changes

### ISSUE THREAD

**Issue by ollcayy** [nvaccess/nvda#11203]
Loadet page when voicing an address suggestion on the NVDA Address bar.
This issue was seen on Chromeium-based edge.
It is not possible to navigate when the page is loaded before the suggestion voiceover ends. It also does not stop speaking when pressed in ctrl mode at the same time. To try this, type the first few letters of a site you have previously visited in the address bar and press' enter. I think this is a problem under Edge. Take a fresh look at NVDA's reporting of edge address suggestions.

### Steps to reproduce:
Open Microsoft edge, enter the first few letters of a site you have previously visited in the address bar and press enter.
### Actual behavior:
NVDA continues to voice options such as press the tab key to search the site x, and the rest of the address x. At this time, the page loads but does not focus on NV.
### Expected behavior:
When NVDA page loads, the address bar related to the address should be spoken. After pressing the enter button, the suggestion has already been accepted and the page is expected to be loaded.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
alpha-20220,1c4519a6 however, the result in 2020.01 is the version
#### Windows version:
Windows 10 20h2 1909
#### Name and version of other software in use when reproducing the issue:
Microsoft edge version83.0.478.37 
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes
2020.01 is the version
#### If addons are disabled, is your problem still occuring?
yes the problem continues
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
yes but problem is note solved

**Comment by josephsl** [comment]
Hi, as much as I wish to deliver you a fresh pizza, without delicious toppings, it becomes hard to make the dinner a more enjoyable one. Please fill out the “toppings” i.e. fill out the template, answer all questions. Thanks.

**Comment by Adriani90** [comment]
I cannot reproduce this with last NVDA alpha and Edge Chromium 83. When typing an address, choosing a suggestion and pressing enter, I can navigate the page imediately by pressing down arrow key and NVDA interupts speech from address bar. Do you have speech interupt for typed characters and speech interupt for enter key enabled in NVDA settings / Keyboard settings?
You need to have both settings enabled.

**Comment by ollcayy** [comment]
Hi,

 

This problem is mostly experienced on chromeium based edge. Character and word speech is currently off.

 

Olcay Aşcı

 <https://certification.nvaccess.org/> NVDA Uzmanlık Sertifikası (NVDA Certified Expert)

Front-end Developer (Arayüz geliştirici)  <https://www.vodafone.com.tr/index.php> Vodafone TR Dijital Varlıklar Müdürlüğü

 <http://eniyideneyim.com/> En İyi Deneyim projesi yöneticisi ve sahibi

Takip edin:

 <https://twitter.com/eniyideneyim> Twitter'de en iyi deneyim  <https://twitter.com/olcayashchi> Twitter'da Olcay Aşcı

 <https://www.linkedin.com/in/olcay-a%C5%9F%C3%A7%C4%B1-b0a619124/> LinkedIn Olcay Aşcı  <https://www.facebook.com/profile.php?id=100000061230801> FaceBook Olcay Aşcı

**Comment by Adriani90** [comment]
These are not the settings I asked for. Interrupt speech for typed characters and interrupt speech for enter key are the settings I mean. Are they on? or off?

Von meinem iPhone gesendet

**Comment by ollcayy** [comment]
“Attempt to cancel speech for expired focus events:  seçim kutusu  Yes  kapalı"

but this setting does not solve this problem.

 

Olcay Aşcı

 <https://certification.nvaccess.org/> NVDA Uzmanlık Sertifikası (NVDA Certified Expert)

Front-end Developer (Arayüz geliştirici)  <https://www.vodafone.com.tr/index.php> Vodafone TR Dijital Varlıklar Müdürlüğü

 <http://eniyideneyim.com/> En İyi Deneyim projesi yöneticisi ve sahibi

Takip edin:

 <https://twitter.com/eniyideneyim> Twitter'de en iyi deneyim  <https://twitter.com/olcayashchi> Twitter'da Olcay Aşcı

 <https://www.linkedin.com/in/olcay-a%C5%9F%C3%A7%C4%B1-b0a619124/> LinkedIn Olcay Aşcı  <https://www.facebook.com/profile.php?id=100000061230801> FaceBook Olcay Aşcı

**Comment by Adriani90** [comment]
@ollcayy are the settings "interrupt speech for typed characters" and "interrupt speech for enter key" enabled or disabled?

**Comment by bhavyashah** [comment]
@ollcayy  This is a friendly reminder to respond to https://github.com/nvaccess/nvda/issues/11203#issuecomment-636387184.

**Comment by Adriani90** [comment]
@ollcayy could you please test with NVDA 2020.1 if this is still occuring?

**Comment by ollcayy** [comment]
Hi,

 

The problem requires a Chromeium-based Microsoft edge.

This does not happen on the original Chrome. When address suggestions are voiced, switching to a different window is not enough to stop the conversation.

The Alt Tab keys will switch to a different window and the ankle sounds will continue to be voiced.

 

Olcay Aşcı

 <https://certification.nvaccess.org/> NVDA Uzmanlık Sertifikası (NVDA Certified Expert)

Digital Product Designer (Dijital Ürün Tasarımcısı)  <https://www.vodafone.com.tr/> Vodafone Türkiye

 <http://eniyideneyim.com/> En İyi Deneyim projesi yöneticisi ve sahibi

Takip edin:

 <https://twitter.com/eniyideneyim> Twitter'de en iyi deneyim  <https://twitter.com/olcayashchi> Twitter'da Olcay Aşcı

 <https://www.linkedin.com/in/olcay-a%C5%9F%C3%A7%C4%B1-b0a619124/> LinkedIn Olcay Aşcı  <https://www.facebook.com/profile.php?id=100000061230801> FaceBook Olcay Aşcı

**Comment by Adriani90** [comment]
I am testing in crhomium based edge 85 and I still cannot reproduce it.
@codeofdusk can you reproduce this? I think this might be related to an addon or so. @ollcayy which addons are you using? Did you try to disable the addons? Did you try with different synthesizers? For example eSpeak or Onecore voices?

**Comment by ollcayy** [comment]
Hi,

 

You can listen to the audio recording showing the problem from the mp3 file below.

I showed it on the record, NVDA cannot be checked until the page load is finished.

NVDA can be recovered if you go to the desktop or switch to a different window.

MY Edge Version: 83

NVDA control gets harder as page loading gets longer.

http://eniyideneyim.com/dosyalar/NVDA-MicrosoftEdgeProblem.mp3

 

Olcay Aşcı

 <https://certification.nvaccess.org/> NVDA Uzmanlık Sertifikası (NVDA Certified Expert)

Digital Product Designer (Dijital Ürün Tasarımcısı)  <https://www.vodafone.com.tr/> Vodafone Türkiye

 <http://eniyideneyim.com/> En İyi Deneyim projesi yöneticisi ve sahibi

Takip edin:

 <https://twitter.com/eniyideneyim> Twitter'de en iyi deneyim  <https://twitter.com/olcayashchi> Twitter'da Olcay Aşcı

 <https://www.linkedin.com/in/olcay-a%C5%9F%C3%A7%C4%B1-b0a619124/> LinkedIn Olcay Aşcı  <https://www.facebook.com/profile.php?id=100000061230801> FaceBook Olcay Aşcı

**Comment by Adriani90** [comment]
Thank you very much for the audio, this clarifies things. In your case it seems NVDA does not allow you to navigate the website by arrow keys after pressing enter on a suggestion in the address bar. It is quite strange because this does not occur to me, I am on the same setup as you are with MS Edge Chromium 83 and Windows 10 2004 update.

When I press enter on a suggestion, NVDA jumps to browse mode and I can navigate as usual.

I think there is something wrong in with the NVDA installation on your PC. Could you download the last alpha version of NVDA and create a portable copy or run it temporary from the installation dialog? Then try to reproduce with the portable copy or with the temporary copy of NVDA. Is the issue still occuring?

**Comment by ollcayy** [comment]
Hi,

 

Problem not solved. NVDA does not respond as the page load time gets longer.

 

Olcay Aşcı

 <https://certification.nvaccess.org/> NVDA Uzmanlık Sertifikası (NVDA Certified Expert)

Digital Product Designer (Dijital Ürün Tasarımcısı)  <https://www.vodafone.com.tr/> Vodafone Türkiye

 <http://eniyideneyim.com/> En İyi Deneyim projesi yöneticisi ve sahibi

Takip edin:

 <https://twitter.com/eniyideneyim> Twitter'de en iyi deneyim  <https://twitter.com/olcayashchi> Twitter'da Olcay Aşcı

 <https://www.linkedin.com/in/olcay-a%C5%9F%C3%A7%C4%B1-b0a619124/> LinkedIn Olcay Aşcı  <https://www.facebook.com/profile.php?id=100000061230801> FaceBook Olcay Aşcı

**Comment by Adriani90** [comment]
@ollcayy are you still having this issue with NVDA 2023.1 and Edge Chromium 112?

**Comment by Adriani90** [comment]
We didn't get an update from the initial author upon request since many years, whether this issue is still reproducible or not. i am closing as abandoned.
If you are still having this issue, Please comment and we can reopen.
